packages/pharos/src/components/toast/pharos-toast.wc.stories.mdx

```diff
@@ -147,3 +147,54 @@ import { GuidelineLink } from '@config/GuidelineLink';
     }}
   </Story>
 </Canvas>
+
+<Canvas>
+  <Story name="Updateable Toast">
+    {() => {
+      const effect = () => {
+        useEffect(() => {
+          document.querySelector('#info-toast-button').click();
+        });
+      };
+      effect();
+      return html`
+        <pharos-button
+          id="info-toast-button"
+          @click="${() => {
+            console.log("click");
```

Comment:
This should instead trigger an action so it can be observed in the **Actions** tab in Storybook.

Reply:
I've been trying to get this to work...but having issues. The modal case is different because it produces and dispatches events. The toast(er) just listens for and consumes events.

Reply:
Fair enough—removing the log here is also fine for the moment.